src/allotropy/parsers/thermo_fisher_qubit_flex/thermo_fisher_qubit_flex_parser.py

```diff
@@ -0,0 +1,432 @@
+
+    Parameters:
+    dataframe (pd.DataFrame): The DataFrame from which to retrieve the value.
+    column (str): The column name from which to retrieve the value.
+    row (int): The row index from which to retrieve the value.
+
+    Returns:
+    Any: The value from the specified cell.
+
+    Raises:
+    AllotropeConversionError: If the value is None.
+    """
+    value = _get_value(dataframe, column, row)
+    if value is None:
+        msg = f"{constants.VALUE_ERROR} '{column}'."
+        raise AllotropeConversionError(msg)
+    return value
+
+
+class ThermoFisherQubitFlexParser(VendorParser):
+    """
+    A class provides the allotrope model of the Thermo Fisher Scientific Qubit Flex files
+    """
+
+    @property
+    def display_name(self) -> str:
+        """
+        Provide the display name of the Adapter
+
+        Returns: the Adapter display name as string
+        """
+        return "Thermo Fisher Scientific Qubit Flex"
+
+    @property
+    def release_state(self) -> ReleaseState:
+        """
+        Provide the Release state of the Adapter
+        """
+        return ReleaseState.WORKING_DRAFT
+
+    def to_allotrope(self, named_file_contents: NamedFileContents) -> Model:
+        """
+        Reads the content of the provided named file and returns it as Model
+
+        Parameters:
+        named_file_contents: contents of the provided named file
+
+        Returns: the Model of the provided named file
+        """
+        return self._get_model(
+            data=ThermoFisherQubitFlexReader.read(named_file_contents),
+            filename=named_file_contents.original_file_name,
+        )
+
+    def _get_model(self, data: pd.DataFrame, filename: str) -> Model:
+        """
+        Reads the content of the provided named file and returns it as Model
+
+        Parameters:
+        data: dataframe of the provided named file
+        filename: original filename of the provided named file
+
+        Returns: the Model of the provided named file
+        """
+        software_version = _get_value(data, "Software Version", (len(data.index) - 1))
+        return Model(
+            field_asm_manifest="http://purl.allotrope.org/manifests/spectrophotometry/BENCHLING/2023/12/spectrophotometry.manifest",
+            spectrophotometry_aggregate_document=SpectrophotometryAggregateDocument(
+                spectrophotometry_document=self._get_spectrophotometry_document(data),
+                data_system_document=DataSystemDocument(
+                    file_name=filename,
+                    software_name=constants.SOFTWARE_NAME,
+                    software_version=software_version,
+                    ASM_converter_name=ASM_CONVERTER_NAME,
+                    ASM_converter_version=ASM_CONVERTER_VERSION,
+                ),
+                device_system_document=DeviceSystemDocument(
+                    model_number=constants.MODEL_NUMBER,
+                    device_identifier="N/A",
```

Comment:
This PR: https://github.com/Benchling-Open-Source/allotropy/pull/490 just added a constant for "N/A" (`from allotropy.parsers.constants import NOT_APPLICABLE`) , please use that.
```suggestion
                    device_identifier=NOT_APPLICABLE,
```

Reply:
Hi @nathan-stender, comments fixed.